pom.xml

```diff
@@ -79,11 +79,11 @@
 	</dependencyManagement>
 
 	<properties>
-		<openMRSVersion>1.10.6</openMRSVersion>
-		<openMRSMinorVersion>1.9</openMRSMinorVersion>
-		<openMRSRuntimeVersion>1.9.12</openMRSRuntimeVersion>
+		<openMRSVersion>2.7.0</openMRSVersion>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<exti18nVersion>1.0.0</exti18nVersion>
+		<exti18nVersion>1.1.0-SNAPSHOT</exti18nVersion>
+		<legacyUiVersion>2.0.0-SNAPSHOT</legacyUiVersion>
```

Comment:
Does this version of the legacyui support Java 8?

Reply:
We dropped support for Java 8 in that version. However, I tested this with Java 8 and got no errors.

Reply:
Of course a few clicks here and there cannot be considered as exhaustive testing. If this pull request is intentionally supporting Java 8, then we should not use a version of a dependency which we know does not support Java 8

Reply:
Downgraded the module to 1.22.0